src/jsi/decorator.h

```diff
@@ -335,18 +335,18 @@ class RuntimeDecorator : public Base, private jsi::Instrumentation {
     plain().instrumentation().collectGarbage();
   }
 
-  bool createSnapshotToFile(const std::string& path) override {
-    return plain().instrumentation().createSnapshotToFile(path);
+  void createSnapshotToFile(const std::string& path) override {
```

Comment:
>void [](start = 2, length = 4)

These seems to be the same as the latest JSI files from Hermes/RN master. Could you please confirm this?

Reply:
Yes, correct, these are the latest headers from hermes/API/jsi